index.html

```diff
@@ -11,5 +11,38 @@
     <title>My Portfolio2</title>
 </head>
 <body>
+  <div class="gen-container">
+    <nav>
+      <h3 class="logo">
+        My Logo
+      </h3>
+      <div class="menu-bars">
+        <img src="header-section\bars.svg" alt="">
+      </div>
+    </nav>
+    <div class="container"> 
+      <h1>
+        I'm Lugard<br>
+        Glad to see <br> 
+        you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you <br>  
+        build a product, feature or website Look<br>  
+        through some of my work and<br>
+        experience! If you like what you see and<br>
+        have a project you need coded, don't <br>
+        hesitate to contact me.
+      </p>
+      <h6>LET'S CONNECT</h6>
+      <div class="images">
+        <img src="header-section\twitter.svg" alt="twitter">&nbsp &nbsp
+        <img src="header-section\linkedin.svg" alt="linkedin">&nbsp &nbsp
+        <img src="header-section\medium.svg" alt="medium">&nbsp &nbsp
+        <img src="header-section\github.svg" alt="github">&nbsp &nbsp
+        <img src="header-section\peace.svg" alt="peace">
+      </div>                          
```

Comment:
- [ ] as the previous reviewer asked you should change the `<div>` tag to the `<section>` tag that will make your code easy to understand and also will improve the SEO for more info check [Mozilla doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)